WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.swift

```diff
@@ -1673,4 +1680,9 @@ extension NotificationsViewController: UIViewControllerTransitioningDelegate {
             self?.tabBarController?.present(alert, animated: true)
         }
     }
+
+    private enum Constants {
+        static let inlineTabAccessCount = 5
```

Comment:
I think we might want to bump this up by 1, as I think it should appear on the 5th appearance of the tab after dismissing the original alert. It currently appears on the 4th. (or change our logic to use `<=` and `>` when checking. That might make more sense.